client/src/Component/navbar/Navbar.js

```diff
@@ -7,35 +7,64 @@ export class Navbar extends Component {
 
   render() {
     return (
-        <div className="Nav">
-            <div className="nav1">
-                <div id="navbar" className="row">
-                    <div className="col col1">
-                        <div className="logo_nav">
-                            <a href="#home">
-                                <img className="logo_kurakoo"  src="./images/kurakoo-logo.png" alt="kurakoo_logo"/>
-                            </a>
-                        </div>
-                    </div>
-                    <div className="col col2">
-                        <div className="seachbar_nav">
-                            <SearchBar />
-                        </div>
-                    </div>
+         // <div className="Nav">
+        //     <div className="nav1">
+        //         <div id="navbar" className="row">
+        //             <div className="col col1">
+        //                 <div className="logo_nav">
+        //                     <a href="#home">
+        //                         <img className="logo_kurakoo"
```

Comment:
We don't want to make changes to the navbar design.

Reply:
How should the navbar look then?
Its the same navbar design as written in Navbar.js.I have just ommitted the unnecessary extra div elements that were added.


Please let me know with a screenshot as to how it should look like!